flutter_local_notifications_windows/lib/src/details/notification_input.dart

```diff
@@ -0,0 +1,114 @@
+import "package:xml/xml.dart";
+
+/// The type of a [WindowsInput].
+enum WindowsInputType {
+  /// A text input.
+  text,
+  /// A multiple choice input.
+  selection,
+}
+
+/// A text or multiple choice input element in a Windows notification.
+abstract class WindowsInput {
+  /// Creates an input field in a notification.
+  const WindowsInput({
+    required this.id,
+    required this.type,
+    this.title,
+  });
+
+  /// A unique ID for this input.
+  ///
+  /// Can be used by buttons to be placed next to this input.
+  final String id;
+
+  /// The type of this input.
+  final WindowsInputType type;
+
+  /// The title of this input.
+  final String? title;
+
+  /// Serializes this input to XML.
+  void toXml(XmlBuilder builder);
+}
+
+/// A text input.
+class WindowsTextInput extends WindowsInput {
+  /// Creates an input field in a notification.
+  const WindowsTextInput({
+    required super.id,
+    this.hintText,
+    super.title,
+  }) : super(type: WindowsInputType.text);
+
+  /// The hint text.
+  final String? hintText;
```

Comment:
Whilst the name isn't wrong and is also what material uses, could this be renamed to `placeholderText` to better align with what the Windows API refers to it as? Presumably you picked this to avoid [placeHolderContent](https://learn.microsoft.com/en-us/uwp/schemas/tiles/toastschema/element-input) per the XML schema with it sounding too generic. If so, the WinRT APIs has it as [placeholderText](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.windows.appnotifications.builder.appnotificationbuilder.addtextbox?view=windows-app-sdk-1.5#microsoft-windows-appnotifications-builder-appnotificationbuilder-addtextbox(system-string-system-string-system-string))

Reply:
Done